sdk/keyvault/azure-security-keyvault-certificates/src/certificate_client_operations.cpp

```diff
@@ -23,7 +23,14 @@ Azure::Response<CertificateOperationProperties> CreateCertificateOperation::Poll
     std::this_thread::sleep_for(period);
   }
 
-  return Azure::Response<CertificateOperationProperties>(
+  if (!m_properties.Error)
+  {
+    auto response = m_certificateClient->GetCertificate(m_properties.Name);
+    m_value = response.Value;
+    m_rawResponse = std::move(response.RawResponse);
+  }
+
+  return Azure::Response<KeyVaultCertificateWithPolicy>(
```

Comment:
so, we are returning this even if `m_properties.Error == true` ??  But only assigning m_value when m_propertie.Error is false?  
What is it going to happen on error?  do we return an empty certificate?  

Shouldn't we throw if Error == true?  and make the error part of the Exception?